vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -111,6 +115,14 @@ ${stderr}`));
                     this.context?.eventStream.post(new CommandExecutionUserCompletedDialogueEvent(`The command ${fullCommandString} failed to run under sudo.`));
                     if(terminalFailure)
                     {
+                        if(error.code === 126)
```

Comment:
I tried and really like how this adds new error handling checks if command invoked cannot execute if user rejects the password prompt. Is it possible to know what `126` refers to in this case?

Reply:
Thanks :) And 126 is what vscode sudo-prompt returns if the user refuses the password request, I dont think there is any other meaning to it besides that, but it might be wise to add a comment stating this